architecture/additional_concepts/authorization.adoc

```diff
@@ -238,6 +273,7 @@ using:
 $ oadm policy reconcile-cluster-roles
 ----
 endif::[]
+ifdef::openshift-origin,openshift-origin,openshift-enterprise,openshift-dedicated[]
```

Comment:
Looks like the double `openshift-origin` is throughout. Not sure if one was meant to be `openshift-online`. If so, consider that the `openshift-online` ifdef is not needed for any `install_config` or `admin_guide` topics.